handlers_test.go

```diff
@@ -663,76 +660,6 @@ func TestHandlers(t *testing.T) {
 	}
 }
 
-func TestCreateLogFilesHandler(t *testing.T) {
```

Comment:
This test provides inconsistent results due to the handler now being asynchronous on opening the fifo. However, I realized that this test is testing the fifo writer, which we already have a test for. With that I decided to delete it.